sipa/login_manager.py

```diff
@@ -1,54 +0,0 @@
-# -*- coding: utf-8 -*-
-from functools import wraps
-
-from flask import request, Blueprint
-from flask_babel import gettext
-from flask_login import LoginManager
-
-
-class SipaLoginManager(LoginManager):
```

Comment:
If you delete things, it's good practice to run a grep through the project so one notices superfluous usages before the CI does.